### DISCUSSION TITLE
Is there any way to collect tweets by hashtags instead of username?

### DISCUSSION THREAD

**Discussion by alarno** [yihong0618/twint#11] [General]
I've been trying and collect tweets for a username works, but, what if I want to collect all tweets with a certain hashtag?

Thank you in advance!!!

Albert

**Comment by yihong0618** [comment]
we can not do not without api caused twitter ban search api for now.
But I will try see if we can get some luck if so I would reply here again.

**Reply by alarno** [reply]
I found in this [wiki](https://github.com/twintproject/twint/wiki/Other-examples), option 3, a way to search a keyword, but I got an error, I'll keep trying, and, if I found something, I'll let you know!
